packages/odie-client/src/components/message/index.tsx

```diff
@@ -58,7 +58,7 @@ const ChatMessage = (
 		sources = [ ...new Map( sources.map( ( source ) => [ source.url, source ] ) ).values() ];
 	}
 
-	const isTypeMessageOrEmpty = ! message.type || message.type === 'message';
+	const isTypeMessageOrEmpty = message.type === 'message' || message.type === 'introduction';
```

Comment:
This change is required to render contact options:
https://github.com/Automattic/wp-calypso/blob/95c5df4b2d69a854c00f762622b1aa2ff5745122/packages/odie-client/src/components/message/index.tsx#L206